index.js

```diff
@@ -503,6 +508,28 @@ module.exports = function (log, indexesPath) {
     })
   }
 
+  function getFullBitset(cb) {
+    ensureIndexSync({ data: { indexName: 'sequence' } }, () => {
+      const bitset = new TypedFastBitSet()
+      const { count } = indexes['sequence']
+      for (var i = 0; i < count; ++i) bitset.add(i)
+      cb(bitset)
+    })
+  }
+
+  function getSeqsBitset(opSeqs, cb) {
+    const offsets = []
+    opSeqs.sort((x, y) => x - y)
+    const opSeqsLen = opSeqs.length
+    const { tarr } = indexes['offset']
+    const tarrLen = tarr.length
+    for (var o = 0; o < tarrLen; ++o) {
```

Comment:
for these, I like the pattern of doing the tarrLen on this line instead of the one above.